lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -134,7 +135,7 @@ def playbook_on_stats(self, stats):
         # Print the timings
         for uuid, result in results:
             msg = ''
```

Comment:
This isn't your code but this isn't necessary.  The next line unconditionally sets msg equal to some other value so this isn't used.